### PR TITLE
Extend Request/Response limit in sql-over-http to 32MiB

### PR DESCRIPTION
**Pull request by nicksrandall**
## Problem
Fixes: #8983

TLDR; We are hitting this limit in more than one app, each with real-world use-cases.

## Summary of changes
This extends the limit from 10MiB to 32MiB (as a sanity check, Planet Scale's limit is currently [64MiB](https://planetscale.com/docs/reference/planetscale-system-limits#query-limits)). I'm open to changing this to whatever you think makes the most sense. 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [x] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

**Comment by nicksrandall** [comment]
My thinking around 32MiB is that it could be an incremental step towards 64MiB (maybe even beyond). You could monitor this change for a period of time and when you are confident that it hasn't had an adverse effect on the system then we can bump it even more. Ideally this limit gets as large as we can make it while still protecting the system.

**Comment by nicksrandall** [comment]
@conradludgate any feedback for this PR? Who would be the right person to review it?

**Comment by conradludgate** [comment]
I will consider this, I'm not 100% sure just yet. Is it possible that you could use the webhooks API for these large queries? I expect the large number of TCP packets needing to be acknowledged would drawf the extra roundtrips for the websocket startup cost.

**Comment by nicksrandall** [comment]
It is not possible for us to switch to websocket for large queries as the queries are authored by our users (not us) so we don’t know before hand the size of result of the query. 

For what it's worth, here are the limits used by other DB providers who offer sql-over-http:
- Supabase: limit is configurable with no hard limit
- Xata: no limit
- Planetscale: 64MiB
- Cloudflare D1: 30 seconds

**Comment by nicksrandall** [comment]
Looks like this PR has been superseded by #9029.
